### PR TITLE
Add minitox to under "Other resources" section in the README

### PR DESCRIPTION
**Pull request by hqwrong**
see https://github.com/TokTok/c-toxcore/pull/874

---
This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/877)

### CODE REVIEW

**Comment by iphydf** [comment]
It should be added to integrations, yes. I have no strong opinion on whether or not it should be linked in the readme as well.

**Comment by tox-user** [comment]
I think it would be nice if people were able to quickly find a simple example use of toxcore without having to search for it on external websites. Especially since docs seem to be outdated. There was an echobot example somewhere but last time I checked it didn't work (not sure from which repo it was).

File: README.md [hunk 1]
```diff
@@ -171,3 +171,5 @@ the API documentation in [toxcore/tox.h](toxcore/tox.h) for more information.
 ### Other resources
 
 - [Another echo bot](https://wiki.tox.chat/developers/client_examples/echo_bot)
+
```
[the remark this answers is not in the document]
Comment (hqwrong) [reply README.md]: Done

**Comment by hqwrong** [comment]
I'll add it to integrations later.

**Comment by iphydf** [comment]
:lgtm_strong:

---

Reviewed 1 of 1 files at r2.
Review status: :shipit: complete! 1 of 1 LGTMs obtained

---



*Comments from [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/877#-:-LExfpyQCBwSugQMBWC_:b-jbsf8z)*

**Comment by iphydf** [comment]
Please enable the checkbox "Allow edits from maintainers." on the bottom right.

Also, rebase on current master.

**Comment by hqwrong** [comment]
strange... I find It's already enabled.

**Comment by iphydf** [comment]
Maybe the problem is that your PR is from master? In the future, try naming your PR branches differently. I'm trying to sign your commit, but github isn't letting me push to your branch.

**Comment by iphydf** [comment]
Problem solved. It was PEBCAK.

**Pull request merged**